docs/pages/eas-update/introduction.mdx

```diff
@@ -15,7 +15,25 @@ All apps running the `expo-updates` library have the ability to receive updates.
 
 Still using Classic Updates? We moved those docs to [the archive](/archive/classic-updates/introduction).
 
-### Get started
+## Pitch
+
+#### Improve User Experience
+
+With EAS Update, you can release hotfixes and enhancements in response to user feedback or market trends, keeping your users engaged and satisfied with your app. Don't wait for the next planned update or manual update request.
+
+#### Broad Compatibility
+
+EAS Update works with vanilla React Native apps, all you need to do is install the lightweight `expo` npm package as a prerequisite, making it a compelling alternative to solutions like [Codepush](/eas-update/codepush).
+
+#### Tailored for Expo
+
+EAS Update is designed for Expo apps, making integration easy and workflows efficient. Use the website UIs to check app status and debug, especially for developers using EAS Build.
+
+#### Customizable Update Strategies
+
+Apply updates through our [expo-updates API](/versions/latest/sdk/updates) and [app.json configuration](/versions/latest/config/app/#updates) if the default behavior is not suitable for you. You’ll always have the upper hand in shaping the update process without compromising your users' experience.
```

Comment:
```suggestion
Apply updates through [`expo-updates` API](/versions/latest/sdk/updates) and [app config configuration](/versions/latest/config/app/#updates) if the default behavior is not suitable for you. You’ll always have the upper hand in shaping the update process without compromising your users' experience.
```

- Since `expo-updates` is a library